portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Constructs marker object. */
+function Marker(lat, lng, title, description) {
+    this.lat = lat;
+    this.lng = lng;
+    this.title = title;
+    this.description = description;
+}
+
+const SYDNEY = new Marker(-33.824032, 151.187674, 'Sydney', 'sydney');
+const NEWYORK = new Marker(40.768093, -73.981904, 'New York City', 'newyork');
+const MANILA = new Marker(14.563149, 121.036559, 'Manila', 'manila');
+const LOS_ANGLES =
+    new Marker(34.105419, -117.706635, 'Los Angeles', 'losangeles');
+
+const CITIES = [SYDNEY, NEWYORK, MANILA, LOS_ANGLES];
+
+const IRVING = new Marker(40.734906, -73.988437, 'Irving Plaza',
+      'Irving Plaza is my favorite small concert venue!');
+const SHAKE_SHACK = new Marker(40.741531, -73.988014, 'Shake Shack',
+      'Shake Shack - my favorite burgers in the world!');
+const LAWN = new Marker(40.780943, -73.966083, 'The Great Lawn',
+      'I love to hang out on the Great Lawn during the summer with friends.');
+const MSG = new Marker(40.749886, -73.993602, 'Madison Square Garden',
+      'The world famous Madison Square Garden, my favorite concert venue.');
+const TRADEWEB = new Marker(40.718845, -74.034609, 'Tradeweb',
+      'I interned at Tradeweb in the summer of 2019');
+const STUY = new Marker(40.717839, -74.013735, 'Stuyvesant High School',
+      'I attended Stuyvesant High School from 2014-2018.');
+const FIDI = new Marker(40.704545, -74.009393, 'Financial District',
+      'I lived in the financial district for almost a year from 2014-2015.');
+const HELLS_KITCHEN = new Marker(40.767702, -73.984881, 'Hell\'s Kitchen',
```

Comment:
Alphabetize these (combined with the names below)

Reply:
I would also rename some of them to make them more obvious. Eg: MSG isn't clear.
Same with STUY.

For some of the actual words: LAWN -> GREAT_LAWN and such; Basically you want descriptive names.